diesel/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "diesel"
-version = "1.4.3"
+version = "1.4.4"
```

Comment:
The current released diesel version is 1.4.5, so we should use that instead

Reply:
Perhaps it would be technically correct, however
a) It makes absolutely no functional difference for GST.
b) It creates unnecessary work as it also mean bumping versions on a lot of `Cargo.toml`s, since the exact versions are used to stop things breaking on almost everything.